starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1894,44 +1898,40 @@ sub print_report {
 		$idpad_n --;
 	    }
             my $acq_prob = "";
+            my $history_blurb = "";
             if ($c->{"TYPE$i"} =~ /BOT|ACQ/){
                 my $prob = $self->{acq_probs}->{$c->{"IMNUM${i}"}};
                 $acq_prob = sprintf("Prob Acq Success %5.3f", $prob);
+
             }
 	    if ($db_stats->{acq} or $db_stats->{gui}){
-	    $table .= sprintf("${idpad}<A HREF=\"%s%s\" STYLE=\"text-decoration: none;\" "
-			      . "ONMOUSEOVER=\"return overlib ('"
-			      . "ACQ total:%d noid:%d <BR />"
-			      . "GUI total:%d bad:%d fail:%d obc_bad:%d <BR />"
-			      . "Avg Mag %4.2f <BR />"
-                              . "$acq_prob"
-			      . "', WIDTH, 220);\" ONMOUSEOUT=\"return nd();\">%s</A>", 
-			      $acq_stat_lookup, $c->{"GS_ID${i}"}, 
-			      $db_stats->{acq}, $db_stats->{acq_noid},
-			      $db_stats->{gui}, $db_stats->{gui_bad}, $db_stats->{gui_fail}, $db_stats->{gui_obc_bad},
-			      $db_stats->{avg_mag},
-			      $c->{"GS_ID${i}"});
-	  }
-	    else{
-                if ((not defined $db_handle) and ($c->{"TYPE${i}"} ne 'FID')){
```

Comment:
The `not defined $db_handle` check looks to be gone now, and I assume that is on purpose.  I actually don't know exactly what that was doing, but were you going to update the `$acq_stat_lookup` to use the kadi server?  I'm not worried about waiting a few more seconds to get the kadi / mica page.  We want to go this direction to sever sybase dependencies.

Reply:
Actually, I think the $db_handle thing here was a mistake.  If I cut that, I think we lose the "free" benefit of only having URLs to click for stars that have history.  I'll review and fix.

I opened a separate issue to update the acq_stat_lookup to use the kadi / mica page and was not planning to hit that here (I'd prefer to make that change when we promote the guide and acq stat lookups through the mica database so everything is fully consistent).

Reply:
If you are sticking with the CGI that does not include AGASC info (which is fine), then leaving this is OK.  For me what I want is the hover-over that tells me if the star has history, plus the always-available link to get the full AGASC info.  Agreed this is some merit to being able to see instantly at a top-level if a star has history, but I think always being able to easily get the AGASC info is more important.

Reply:
Good point.  I'll add that to requirements in the issue for converting to the kadi star lookups.